metrics.py

```diff
@@ -149,7 +149,9 @@ def get_stats(self):
                 errors = plugins_metadata['errors']
                 first_failed = sorted(errors.keys())[0]
                 plugins['failed_plugin'] = first_failed
-                plugins['exception'] = errors[first_failed].split("(")[0].replace(',', ' ')
+                exception_text = errors[first_failed].split("(")[0]
+                # Put exception text in quotes to allow commas in the exception
+                plugins['exception'] = '"%s"' % exception_text
```

Comment:
Without putting backslashes before quotes, CSV parsing would break. e.g.

```
foo,"bar"baz","bak"bou"
```
is parsed as:
 - `foo`
 - `barbaz"`
 - `bakbou"`

What do you mean it replaces quotes with double-quotes? I don't see that here. The string you've shown above was transformed from `unbalanced " quote` into `"unbalanced \" quote"` i.e. it was quoted, and double-quotes inside the string were escaped, as expected.